.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,13 @@
+name: shellcheck
+on: [push, pull_request]
```

Comment:
```suggestion
on:
  workflow_dispatch:
  push:
    branches:
      - main
      - master
  pull_request:
```

Reply:
Sorry for the slow reply.

Done, but I've removed `master` now too. Please click approve again it looks good.